### ISSUE TITLE
[BUG] when LastEvaluatedKey is null function should return latest items

### ISSUE THREAD

**Issue by danishjamal104** [code-gambit/VT-lambdas#74]
**Describe the bug**
In `lambda_for_fileList` when we pass empty string or null as `LastEvaluatedKey` query parameter then 
```
{
  "statusCode": 500,
  "error": "Internal Server Error"
}
``` 
is returned, instead of when the error occurred while parsing`LastEvaluatedKey` latest item should be returned without throwing a server error.

Area Where Bug Occurred
- [x] Lambda <@lambda_for_fileList>
- [ ] DynamodDb

**How to reproduce the error/bug**
<img width="1174" alt="Screenshot 2021-05-26 at 8 06 37 PM" src="https://user-images.githubusercontent.com/31315800/119679315-e3eb0b80-be5d-11eb-983b-19214927f4dc.png">

**Before submitting an issue check the below**
- [x] I have synced all my node versions as mentioned in the project"
- [x] I am using the same version of npm as is the project

**Comment by lakshya-20** [comment]
This issue has been resolved by me #76. you can close this now.

**Comment by danishjamal104** [comment]
Sure
